### PR TITLE
feat(Segment): add `size` prop

### LINKED ISSUE #568: feat(Segment): add `size` prop
The Segment `size` prop is not documented in the primary SUI docs, however, it is supported.

https://github.com/Semantic-Org/Semantic-UI/blob/master/dist/components/segment.css#L752

We should add the `size` prop to the Segment.

### PR DESCRIPTION
**Pull request by layershifter**
Fixes #568.

I've added prop and tests, do we need add docs?

### CODE REVIEW

File: src/elements/Segment/Segment.js [hunk 1]
```diff
@@ -78,10 +81,11 @@ Segment._meta = {
   type: META.TYPES.ELEMENT,
   props: {
     attached: ['top', 'bottom'],
+    color: SUI.COLORS,
     floated: SUI.FLOATS,
-    textAlign: SUI.TEXT_ALIGNMENTS,
     padded: ['very'],
-    color: SUI.COLORS,
+    size: _.without(SUI.SIZES, 'medium'),
```
Comment (layershifter) [inline src/elements/Segment/Segment.js:87]: There are no `medium` size.

**Comment by levithomason** [comment]
Docs would be fantastic!

**Comment by layershifter** [comment]
> Docs would be fantastic!

Added examples 👍

**Comment by levithomason** [comment]
Awesome, thanks a million @layershifter.  Released in `stardust@0.48.3`.

**Pull request merged**